src/clr-core/tag/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "ngPackage": {
+    "lib": {
+      "entryFile": "tag.element.ts"
```

Comment:
Can we have this point to `index.ts`? Also in the package.json we need to add a new side effect entrypoint. `"./**/clr-core-tag.js"`

https://github.com/vmware/clarity/blob/master/src/clr-core/package.json#L11